packages/zowe-explorer-api/src/profiles/AuthHandler.ts

```diff
@@ -40,7 +40,7 @@ export type ProfileLike = string | imperative.IProfileLoaded;
 export class AuthHandler {
     private static authPromptLocks = new Map<string, Mutex>();
     private static profileLocks = new Map<string, Mutex>();
-    private static enabledProfileTypes: Set<string> = new Set(["zosmf"]);
+    private static enabledProfileTypes: Set<string> = new Set(["zosmf", "zftp"]);
```

Comment:
Dumb question: Is this something we will need to do for other extenders?

Reply:
Extenders can enable locks for their profile types using the `AuthHandler.enableLocksForType` function, but I've kept the profile types to just the ones we maintain to avoid breaking changes. 

If we want to in v4, we could remove the check entirely and use the profile locks for all types 😋

Reply:
I would always be in favor of consistency. 😋 
But C4 sounds good 👍

Reply:
since we use the zFTP extension as a sample should we have the zFTP extension share how extenders are to enable the lock.  Having the zFTP ext make the `AuthHandler.enableLocksForType` call?